pjesak.pde

```diff
@@ -21,41 +30,51 @@ class Pjesak implements Collideable{
     y1 = (float) (level.centerY(tmpY) - h / 2);
     x2 = (float) (level.centerX(tmpX + tmpW - 1) - w / 2);
     y2 = (float) (level.centerY(tmpY + tmpH - 1) - h / 2);
-    distance = dist(x1, y1, x2, y2); // duljina pjesakovog puta
+    distance = dist(x1, y1, x2, y2); // udaljenost od (x1, y1) do (x2,y2)

```

Comment:
Pa piše da je udaljenost u kodu. Možda "duljina pješakovog puta je udaljenost od točke (x1, y1) do (x2, y2)" ili tako nešto.

Reply:
Objašnjenje je vezano uz funkciju dist.